packages/docs/docs/billing/insurance-eligibility-checks.md

```diff
@@ -0,0 +1,135 @@
+import ExampleCode from '!!raw-loader!@site/..//examples/src/billing/insurance-eligibility-checks.ts';
+import MedplumCodeBlock from '@site/src/components/MedplumCodeBlock';
+
+# Insurance Eligibility Checks
+
+Insurance eligibility checks determine whether a patient's insurance is active, in-network, and has applicable benefits. They ensure that the provider will ultimately get compensated by the patient's insurer for a specific product or service.
+
+## Use Cases
+
+Insurance eligibility checks cover a variety of use cases, but three of the most important are:
+
+1. Requests for prior authorization
+2. Confirmation of benefits
+3. Coordination of care
```

Comment:
Perhaps: Confirm that insurance policy covers a specific [service type](https://x12.org/codes/service-type-codes)